ui/src/views/404/index.jsx

```diff
@@ -23,7 +24,7 @@ export default ({ beaconUrl }: Object) =>
         </div>
         <Logo
             block-styles={{ guardian: { fill: colour.brandBlueDark } }}
-            width={250}
+            svg-styles={logo}
```

Comment:
what happened if you just used `style` here as normal? did it not work?

Reply:
It did not, for reasons I was unable to ascertain because it was not possible to easily debug the `svg-loader`. I may have found a way to debug now, so let's revisit